src/main/java/seedu/address/logic/commands/AvailableCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_GROUP;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.TutorialTagContainsGroupPredicate;
+import seedu.address.model.tag.TutorialTag;
+
+/**
+ * Finds and lists all available TAs for the tutorial group.
+ */
+public class AvailableCommand extends Command {
+    public static final String COMMAND_WORD = "available";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all available TAs for the tutorial slot.\n"
+            + "Parameters: [FLAG] [TUTORIAL SLOT] \n"
```

Comment:
For consistency with other commands should change to
"Parameters: available /g TUTORIAL"
(using COMMAND_WORD and PREFIX_GROUP)

Reply:
Thank you will take note of that 👍